docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -67,6 +78,16 @@ applies the update to nodes according to the `UpdateConfig` policy:
     redis
     ```
 
+    The scheduler applies rolling updates as follows:
+
+    * Stop the initial number of tasks according to `--update-parallelism`.
+    * Schedule updates for the stopped tasks.
+    * Start the containers for the updated tasks.
+    * Wait for the health checks on individual tasks to return a state of either
```

Comment:
I believe it does wait for the health checks to return `RUNNING` or `FAILED`. But it doesn't care about the result - it just keeps going either way :(